<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaData</class>
 <widget class="QWidget" name="MetaData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>400</height>
      </size>
     </property>
     <property name="title">
      <string>Meta Information</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Deck Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QLineEdit" name="deckName">
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="placeholderText">
         <string>Deck Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QLineEdit" name="deckAuthor">
        <property name="placeholderText">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="6">
       <widget class="QPlainTextEdit" name="deckDescription"/>
      </item>
      <item row="3" column="5" colspan="2">
       <widget class="QLineEdit" name="deckVersion">
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="placeholderText">
         <string>Version Number</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Deck Version:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Creation Date:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="date">
        <property name="text">
         <string>21/7/2007</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>API Version:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="api">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="buttonReset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="icon">
         <iconset resource="../deckeditor.qrc">
          <normaloff>:/DeckEditor/Resources/Icons/arrow-curve-180-left.png</normaloff>:/DeckEditor/Resources/Icons/arrow-curve-180-left.png</iconset>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="buttonSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="../deckeditor.qrc">
          <normaloff>:/DeckEditor/Resources/Icons/disk-return-black.png</normaloff>:/DeckEditor/Resources/Icons/disk-return-black.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../deckeditor.qrc">:/DeckEditor/Resources/Icons/information.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>You can edit the Deck's Meta Information, such as Name, Author and Version here.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../deckeditor.qrc"/>
  <include location="../deckeditor.qrc"/>
 </resources>
 <connections/>
</ui>
